rebar.config

```diff
@@ -34,6 +34,7 @@
                    %% upstream version:
                    %% https://github.com/massemanet/redbug/pull/10
                  , [ {redbug, {git, "https://github.com/robertoaloi/redbug.git", {branch, "relax-beam-lib-error-matching"}}}
+                   , {eflame, {git, "https://github.com/jfacorro/eflame.git", {branch, "various.improvements"}}}
```

Comment:
I would prefer to bring in the upstream version, if possible.
What are the included changes?
Are you pull-requesting them to upstream?

Reply:
This is the PR with all the [improvements](https://github.com/proger/eflame/pull/17).